src/js/overlay.js

```diff
@@ -1,5 +1,5 @@
 import Delegate from 'ftdomdelegate';
-import viewport from '@financial-times/o-viewport';
+import viewport from 'o-viewport';
```

Comment:
This shouldn't work as `o-viewport` on npm is in the `@financial-times` namespace